src/tm_devices/commands/gen_e3e9uu_lpdmso/display.py

```diff
@@ -1488,9 +1516,9 @@ def scale(self) -> DisplayWaveview1RfMagnitudeItemVerticalScale:
             ```
 
         Info:
-            - ``1`` is always WAVEView1.
-            - ``RF_MAGnitude<x>`` specifies the number of the Magnitude vs. Time Frequency waveform.
```

Comment:
This got put into another description when it should have stayed on its own line